### PR TITLE
Remove legacy api endpoints

### PR DESCRIPTION
**Pull request by fkneeland-figure**
## Description

In this pr I removed all of the deprecated api endpoints from swagger.  I also updated the make file and documentation to help future contributors to swagger in provenance have an easier time dealing with build/caching issues when trying to test changes to swagger.

closes: #380

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by fkneeland-figure** [comment]
@iramiller @dwedul-figure would you two be able to review this when you have a minute?  Thanks!

File: Makefile [hunk 1]
```diff
@@ -493,7 +493,7 @@ proto-update-deps:
 ### Docs
 ##############################
 update-swagger-docs: statik
-	./scripts/protoc-swagger-gen.sh
+	make proto-swagger-gen
```
Comment (dwedul-figure) [inline Makefile:496]: I'm not too well versed in Makefile syntax, but I think this is better handled by putting the other action to do on the main action line.
```
update-swagger-docs: statik proto-swagger-gen
```

Comment (iramiller) [reply Makefile:496]: Indeed you chain targets instead of nesting contexts like this... refer to examples such as `proto-all`

Comment (fkneeland-figure) [reply Makefile]: 👍 updated @dwedul-figure @iramiller

**Review by dwedul-figure — APPROVED** [review]
LGTM

**Pull request merged**